ab2d-filters/src/main/java/gov/cms/ab2d/filter/EOBLoadUtilities.java

```diff
@@ -25,12 +25,15 @@ public class EOBLoadUtilities {
      *             Explanation of Benefit data retrieved from Blue
      * @return the ExplanationOfBenefit object
      */
-    public static org.hl7.fhir.dstu3.model.ExplanationOfBenefit getSTU3EOBFromFileInClassPath(String fileInClassPath) {
+    public static org.hl7.fhir.dstu3.model.ExplanationOfBenefit getSTU3EOBFromFileInClassPath(
+            String fileInClassPath) {
```

Comment:
I am not sure I like this particular formatting. I would prefer to move the closing parenthese on the function down

getSTU3EOBFromFileInClassPath(
    String fileInClassPath
) {
...
}

At least that is how I am more used to seeing it...but thats also languages other than Java. 🤷